.travis.yml

```diff
@@ -76,3 +75,31 @@ script:
   - ./bin/dxc -T ps_6_0 -Fo passthru-ps.spv ../tools/clang/test/CodeGenSPIRV/passthru-ps.hlsl2spv -spirv
   - ./bin/clang-spirv-tests --spirv-test-root ../tools/clang/test/CodeGenSPIRV/
   - ./bin/clang-hlsl-tests --HlslDataDir $PWD/../tools/clang/test/HLSL/
+
+after_success:
+  # Create tarball for deployment
+  - if [[ "${CC}" == "clang" ]]; then
+      export ARTIFACTS_DIR=build-${TRAVIS_OS_NAME}-${DXC_BUILD_TYPE};
+      cd .. && mkdir ${ARTIFACTS_DIR};
+      cd ${ARTIFACTS_DIR} && mkdir bin && mkdir lib && mkdir include && cd ..;
+      cp build/lib/libdxcompiler.so.3.7 ${ARTIFACTS_DIR}/lib/;
+      cp build/bin/dxc ${ARTIFACTS_DIR}/bin/;
```

Comment:
On my Linux builds, dxc is just a symbolic link to the actual executable "dxc-3.7" The 3.7 is a relic of the clang build which is version 3.7. That's something I've considered changing.

Reply:
yeah I think that'd be a good change.

Reply:
In the meantime, I think this needs to copy from dxc-3.7. If you want to copy it to an executable called dxc, that's fine with me.

Reply:
I think `cp` actually copies the underlying dxc-3.7. At least that's what happens when I try it on my machine

Reply:
hah! So it does! I didn't think I had anything left to learn about ```cp```!

Thanks for the TIL!